app/views/idv/phone_errors/warning.html.erb

```diff
@@ -1,4 +1,68 @@
-<%= render('idv/phone_errors/warning') do %>
-  <p><%= t('idv.failure.phone.warning') %></p>
-  <p><strong><%= t('idv.failure.attempts', count: @remaining_attempts) %></strong></p>
-<% end %>
+<%= render(
+      'idv/shared/error',
+      type: :warning,
+      title: t('titles.failure.phone_verification'),
+      heading: t('idv.failure.phone.warning.heading'),
+      current_step: :verify_phone_or_address,
+      options: [
+        decorated_session.sp_name && {
+          url: return_to_sp_failure_to_proof_path(
+            step: 'phone',
+            location: local_assigns.fetch(:name, 'warning'),
+          ),
+          text: t('idv.troubleshooting.options.get_help_at_sp', sp_name: decorated_session.sp_name),
+          new_tab: true,
+        },
+      ].select(&:present?),
+    ) do %>
+
+      <p>
+        <%= t('idv.failure.phone.warning.you_entered') %>
+        <strong><%= PhoneFormatter.format(@phone, country_code: @country_code) %></strong>
```

Comment:
My understanding from UX shareout was that this could be a list of phone numbers if they fail with multiple ones. Is that included in this ticket?

Reply:
The banner shouldn't be there when going back to retry, so I'm thinking that needs to be fixed. I think it would also be good to avoid pre-filling the number, but those two things weren't explicitly included in the ticket. @benjaminchait should we do this here or create a new ticket for this work?

And the list of phone numbers isn't included on this ticket.

Reply:
I think that is a separate ticket. The designs for this only have the 1 number.